src/MentionScrollBar.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+const MentionScrollBar = ({
+  active,
+  className,
+  item: Item,
+  options,
+  hoverItem,
+  selectItem,
+  style = {}
+}) => {
+  const menuStyle = {
+    ...style,
+    top: '0',
+    width: '100%',
+    display: 'flex',
+    flexDirection: 'row'
+  };
+
+  return (
+    <div style={menuStyle} className={className}>
+      {options.map((option, idx) => {
+        return (
+          <div key={idx} onClick={selectItem(idx)} onMouseOver={hoverItem(idx)}>
+            <Item active={active === idx} {...option} />
+          </div>
+        );
+      })}
+    </div>
+  );
+}
+
+export default MentionScrollBar;
```

Comment:
Can we see if we can pull the component out? (allow a custom component to be passed into githubish mentions) it would keep things cleaner and more generic. if its too difficult or may take too long then dont worry about it i guess..